Sources/Purchasing/EntitlementInfos.swift

```diff
@@ -31,8 +31,15 @@ import Foundation
         return self.all[key]
     }
 
+    // TODO: public
+    internal let validationResult: HTTPResponseValidationResult
```

Comment:
we should check with product. the original name for this was "trusted", values boolean (potentially optional to represent the not requested state)